packages/components/src/components/mobile-carousel/mobile-carousel.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import classNames from 'classnames';
+import Icon from '../icon';
+
+export const MobileCarousel = ({ children }) => {
+    const carousel_body = React.createRef();
+    const [min_width, setMinWidth] = React.useState(0);
+    const [index, setIndex] = React.useState(0);
+    const [total_elements, setTotal] = React.useState(0);
+
+    // Carousel is only active for more than 1 child items
+    if (children.length <= 1) {
+        return children;
+    }
+
+    React.useEffect(() => {
+        setMinWidth(carousel_body.current.children[0].clientWidth + 1);
+        carousel_body.current.scrollLeft = index * (min_width + 1);
+        setTotal(React.Children.count(children));
+    });
+
+    const goPrev = () => {
+        if (index !== 0) {
```

Comment:
Is `touchSwipe` or `touchDrag` handled here for switching to different slides in the carousel? Since this is a mobile only component, its good to have touch capabilities for good UX.

Reply:
Not in this PR, there is another card for it to enable swipe.